Library/Homebrew/cmd/update-report.rb

```diff
@@ -235,20 +235,22 @@ def output_update_report
       EOS
     end
 
-    return if new_repository_version.blank?
+    return if new_tag.blank? || args.quiet?
 
     puts
-    ohai "Homebrew was updated to version #{new_repository_version}"
-    Settings.write "latesttag", new_repository_version
-    if new_repository_version.split(".").last == "0"
+    ohai "Homebrew was updated to version #{new_tag}"
+    puts <<~EOS
+      Changelogs for all Homebrew versions can be found at:
+        #{Formatter.url("https://github.com/Homebrew/brew/releases")}
+    EOS
+
+    new_major_version, new_minor_version = (new_tag.split(".")[0, 2]).map(&:to_i)
+    old_major_version, old_minor_version = (old_tag.split(".")[0, 2]).map(&:to_i) if old_tag.present?
+    if old_tag.blank? || (new_major_version > old_major_version) \
+        || (new_minor_version > old_minor_version)
       puts <<~EOS
         More detailed release notes are available on the Homebrew Blog:
```

Comment:
```suggestion
        More detailed release notes for major and minor releases are available on the Homebrew Blog:
```

I think this wording is better since it's no longer true that the most recent version is major/minor